tests/integration_tests/test_saas_task.py

```diff
@@ -198,3 +199,198 @@ def test_saas_erasure_request_task(
         f"{dataset_name}:conversations": 0,
         f"{dataset_name}:messages": 0,
     }
+
+
+@pytest.mark.integration_saas
```

Comment:
Can we also add `@pytest.mark.saas_connector_sentry` so we can target just Sentry should we need to

Reply:
yes good idea